src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,17 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        List<Integer> allNumbers = numbers
+                .stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
```

Comment:
```suggestion
        List<Integer> allNumbers = numbers.stream()
                .flatMap(s -> Arrays.stream(s.split(",")))
```
It would be better not place stream() in new line.

Reply:
fixed